scripts/get-serve-d.js

```diff
@@ -0,0 +1,26 @@
+// script to download serve-d binaries from GitHub
+
+const downloadRelease = require("@terascope/fetch-github-release")
+const { join, dirname, basename, extname } = require("path")
+const { execSync } = require("child_process")
+const decompress = require("decompress")
+const decompressTarxz = require("decompress-tarxz")
+
+;(async () => {
+  const distFolder = join(dirname(__dirname), "dist")
+  execSync(`shx rm -rf ${distFolder}`)
```

Comment:
could this break if __dirname has a space in it? I think with variables it's always safer to instead use the overload using arrays

Also instead of invoking a command for rm -rf, it might be better to use a package like rimraf for compatibility

Reply:
> could this break if __dirname has a space in it? I think with variables it's always safer to instead use the overload using arrays

No. I am using `path` functions. Node's path functions are able to handle spaces.

> Also instead of invoking a command for rm -rf, it might be better to use a package like rimraf for compatibility

I am using [`shx`](https://github.com/shelljs/shx) which is portable shell based for Node, and it is better than rimraf, and support many more commands.

Reply:
it's not the path function that would break with spaces, it would be the `rm` command that would break with spaces

Reply:
Oh. OK. I used shelljs directly to fix this issue.
https://github.com/Pure-D/atomize-d/pull/34/commits/442d91c136a2883949d5da775dced67c9fcc2169